### PR TITLE
Fixes for QGis, Preparation for GlideSnow Connection [com1DFA]

### PR DESCRIPTION
**Pull request by fso42**
- Fixes for QGIS
- New com5GlideSnow module
- Preparation for Connector

### CODE REVIEW

File: avaframe/com1DFA/damCom1DFA.pyx [hunk 1]
```diff
@@ -438,8 +438,8 @@ def initializeWallLines(cfg, dem, wallLineDict, savePath=''):
   cdef double w[4]
   cdef int Lx0, Ly0
   if wallLineDict is not None:
-    log.info('Initializing dam line from: %s' % str(wallLineDict['fileName'][0]))
-    log.info('Dam line feature: %s' % str(wallLineDict['Name'][0]))
+    log.debug('Initializing dam line from: %s' % str(wallLineDict['fileName'][0]))
+    log.debug('Dam line feature: %s' % str(wallLineDict['Name'][0]))
     # get z coordinate of the dam polyline
```
Comment (awirb) [inline avaframe/com1DFA/damCom1DFA.pyx:443]: Is there still some info that a dam is initialized and used in the log if all these are set to debug? I think one info like for the secondary release areas would be good

File: avaframe/runGlideSnowCom1DFA.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    else:
+        avalancheDir = cfgMain['MAIN']['avalancheDir']
+
+    # Start logging
+    log = logUtils.initiateLogger(avalancheDir, logName)
+    log.info('MAIN SCRIPT')
+    log.info('Current avalanche: %s', avalancheDir)
+
+    # ----------------
+    # Clean input directory(ies) of old work and output files
```
Comment (awirb) [inline avaframe/runGlideSnowCom1DFA.py:57]: actually the delete output flag is set to False

File: avaframe/runGlideSnowCom1DFA.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    _, plotDict, reportDictList, _ = glideSnow.runGlideSnowTool(cfgMain, glideSnowCfg)
+
+    # Get peakfiles to return to QGIS
+    avaDir = pathlib.Path(avalancheDir)
+    inputDir = avaDir / 'Outputs' / 'com1DFA' / 'peakFiles'
+    peakFilesDF = fU.makeSimDF(inputDir, avaDir=avaDir)
+
+    # ----------------
+    # Collect results/plots/report  to a single directory
+    # make simple plots (com1DFA, com2AB)
```
Comment (awirb) [inline avaframe/runGlideSnowCom1DFA.py:75]: com2AB?

Comment (awirb) [reply avaframe/runGlideSnowCom1DFA.py:75]: where are the plots made?

File: avaframe/runGlideSnowCom1DFA.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+    Parameters
+    ----------
+    avalancheDir: str
+        path to avalanche directory (setup eg. with init scipts)
+
+    Returns
+    -------
+    abShpFile: str
+        path to com2AB results as shapefile
```
Comment (awirb) [inline avaframe/runGlideSnowCom1DFA.py:32]: remove?

**Review by awirb — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/damCom1DFA.pyx (hunk 1 above)
Comment (fso42) [reply avaframe/com1DFA/damCom1DFA.pyx:443]: mh yes, but it needs to be done outside of the pyx file. I'll start an issue...

**Comment by fso42** [comment]
Standardtests ok

**Pull request merged**